generators/server/templates/src/main/java/package/config/_OAuth2ServerConfiguration.java

```diff
@@ -85,6 +91,8 @@ public void configure(HttpSecurity http) throws Exception {
             .and()
                 .csrf()
                 .disable()
+                // By default CORS is disabled. Uncomment here and in application.yml to enable.
+                // .addFilterBefore(corsFilter, UsernamePasswordAuthenticationFilter.class)
```

Comment:
Two options I see are:
 - Always register the filter, just don't apply CORS headers if not enabled.  It was made conditional due to a microservice issue in #3318 but now all CORS config is handled by the gateway since #3575.
 - Remove all the code except the comment and add a note for the user to add CorsFilter to the constructor (easier)